combat-trainer.lic

```diff
@@ -1979,6 +1982,11 @@ class GameState
     weapon_skill == 'Brawling'
   end
 
+  def brawl_tactics?
+    return true if !brawling?
+    ((weapon_skill == 'Brawling') && (weapon_name == ''))
```

Comment:
Do you need to check `weapon_skill` here if you checked `brawling?` on the line above?